src/test_coinbase.cairo

```diff
@@ -0,0 +1,98 @@
+use shinigami::transaction::TransactionTrait;
+use shinigami::transaction::Transaction;
+use shinigami::transaction::TransactionOutput;
+use shinigami::transaction::TransactionInput;
```

Comment:
Can you combine these into something like `use shinigami::transaction::{TransactionTrait, Transaction, ...}`?